templates/sios-datakeeper-master.template

```diff
@@ -801,15 +734,7 @@
                         "Ref": "QSS3BucketName"
                     },
                     "QSS3KeyPrefix": {
-                        "Fn::Join": [
-                            "/",
-                            [
-                                {
-                                    "Ref": "QSS3KeyPrefix"
-                                },
-                                "submodules/quickstart-microsoft-activedirectory"
-                            ]
-                        ]
+                        "Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-activedirectory"
```

Comment:
Needs a slash at the end:
"Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-activedirectory/"